src/current/v23.1/live-migration-service.md

```diff
@@ -0,0 +1,646 @@
+{% endcomment %}
+
+To perform a consistent cutover with the LMS:
+
+{{site.data.alerts.callout_info}}
+These steps assume you have already followed the overall steps to [prepare for migration]({% link {{ page.version.version }}/migration-overview.md %}#prepare-for-migration). In particular, [update your schema and application queries]({% link {{ page.version.version }}/migration-overview.md %}#update-the-schema-and-queries) to work with CockroachDB.
+{{site.data.alerts.end}}
+
+1. [Configure the LMS](#configuration) with your deployment details, and follow our [security recommendations](#security).
+
+1. Set the shadowing mode to [`none`](#none).
+
+	{{site.data.alerts.callout_danger}}
+	Do not use the [`sync`](#sync) or [`strict-sync`](#strict-sync) shadowing modes when performing a consistent cutover. Data correctness and consistency cannot be guaranteed in these configurations.
+	{{site.data.alerts.end}}
+
+1. Set up ongoing replication between the source database and CockroachDB, using a tool such as [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}).
+
+1. Send application requests to the LMS, which routes the traffic to the source database. The source database is designated the source of truth.
+
+1. Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate that the replicated data on CockroachDB is consistent with the source of truth.
+
+1. Begin the consistent cutover. **Requests are now queued in the LMS**, including queries from existing connections and new connection requests to the LMS:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt-lms-cli cutover consistent begin {flags}
+	~~~
+
+	This command tells the LMS to pause all application traffic to the source of truth. The LMS then waits for transactions to complete and prepared statements to close.
+
+1. Verify that replication on CockroachDB has caught up with the source of truth. For example, write a marker record to the source database and check that it exists on CockroachDB.
+
+	If you have an implementation that replicates back to the source database, this should be enabled before committing the cutover.
+
+1. Once all writes have been replicated to the target database, commit the consistent cutover:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	molt-lms-cli cutover consistent commit {flags}
+	~~~
+
+	This command tells the LMS to switch the source of truth to the target database. Application traffic is now routed to the target database, and requests are processed from the queue in the LMS.
+
+	To verify that CockroachDB is now the source of truth, you can run `molt-lms-cli status`.
+
+1. Again, use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate that the data on the source database and CockroachDB are consistent.
+
+If any problems arise during a consistent cutover:
```

Comment:
I think that we probably need a "common problems" section here, where we instruct users how to detect and remedy problems with the LMS.  I think @ZhouXing19 is taking a first crack at this.

Reply:
yes, i have drafted a troubleshooting section and will discuss with oliver today.

Reply:
The troubleshooting doc is ready for review (got LGTM from Oliver): https://docs.google.com/document/d/1zrL_O--8l9tjYyGIIKvW_U4viiG07XRnOaZtlC3R1u4/edit
Also added it to the runbook: https://cockroachlabs.atlassian.net/wiki/spaces/MIG/pages/3162506994/Live+Migration+Service+LMS+Runbook

Reply:
I will have to save this for a separate DOC issue. @cotedm when you have a chance, do you mind filing one and outlining exactly what we should cover (if not everything in the linked doc)?